tools/provision/lib.sh

```diff
@@ -368,6 +368,13 @@ function install_libaptpkg() {
   fi
 }
 
+function link_openssl_pkgconfig_files() {
+  BREW_PREFIX=$(brew --prefix)
```

Comment:
Will existing OpenSSL installs, via brew-- which wont run this, break on linking?